tests/fixtures/require-space-after-keywords/good/example.js

```diff
@@ -21,12 +21,12 @@ switch (x) {
 
 try {
   x();
-} catch(error) {
+} catch (error) {
```

Comment:
So you want spaces after all keywords, but not catch? Just trying to understand.